app/controllers/service_controller.rb

```diff
@@ -33,9 +33,20 @@ def button
       service_retire
     when 'service_retire_now'
       service_retire_now
+    when "custom_button"
+      @display == 'generic_objects' ? generic_object_custom_buttons : custom_buttons
```

Comment:
Should we raise here in an else case with a generic "not supported" warning?  I assume before this change, it didn't do anything when you clicked the button.  What's the best way to handle a button with no defined action?

Reply:
> Should we raise here in an else case with a generic "not supported" warning? I assume before this change, it didn't do anything when you clicked the button. What's the best way to handle a button with no defined action?

I had added a default action.
<img width="654" alt="image" src="https://github.com/ManageIQ/manageiq-ui-classic/assets/87487049/1bb403a3-0499-40ee-aff2-7e6c42b3e436">